dam/loans/views.py

```diff
@@ -21,6 +22,7 @@ def reservations(request, reservation_id):
         if "Approve" in request.POST:
             ItemLoan.objects.create(item=reservation.item, client=reservation.client, approved_by=request.user)
```

Comment:
```suggestion
            ItemLoan.objects.create(item=reservation.item, client=reservation.client, approved_by=request.user, due_on=timezone.now() + timezone.timedelta(days=14))
```
This should be accurate from when the Reservation is approved.